open-company/open-company-web
# Column chart

card: https://trello.com/c/h2fomlmY

test:
- open a report in edit mode
- make sure you have data in the headcount and in the compensation sections
- check the column charts: they should be shown with blue columns
